### PR TITLE
Add a preliminary test for Studio

### PR DESCRIPTION
**Pull request by sdnts**
Adds a test that checks if Studio's server is up. Doesn't check if Studio displays data correctly (yet)

### CODE REVIEW

File: src/packages/migrate/src/Studio.ts [hunk 1]
```diff
@@ -22,11 +23,18 @@ export class Studio {
...
   private port?: number
-
-  constructor({ schemaPath, browser, port }: StudioOptions = {}) {
+  private staticAssetDir?: string
+
+  constructor({
+    schemaPath,
+    browser,
+    port,
+    staticAssetDir,
```
Comment (madebysid) [inline src/packages/migrate/src/Studio.ts:32]: Added this option to Studio so I can override it from tests.

File: src/packages/migrate/src/__tests__/Studio.test.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    `,
+    )
+    // For the time being, it is okay that the SQLite file used in this schema doesn't exist
+    // This is because the test only sees if Studio loads, and not that data is shown correctly (for now)
+  })
+
+  beforeEach(() => {
+    studioInstance = new Studio({
+      schemaPath: path.resolve(`${testRootDir}/schema.prisma`),
+      staticAssetDir: path.resolve('../../build/public'),
```
Comment (madebysid) [inline src/packages/migrate/src/__tests__/Studio.test.ts:44]: Is it okay to depend on the `build/public` directory? Gives me the heebie jeebies, but I can't think of another way of doing this.

**Comment by madebysid** [comment]
@timsuchanek I have a couple of questions:

1. When I ran `pnpm run setup`, a few `package.json` files changed, is this normal? I've put these changes in a separate commit, so I can revert if necessary.

2. The test seems to be failing (during client generation) because of some Cloudfront issue. I don't think I understand the problem so I didn't try antod debug too much. 
That being said, I noticed that the version of the `migrate` package in its `package.json` is quite old, so I tried updating it to the latest `dev`, which didn't help (I don't think I should be doing it manually anyway). What can I do to fix this?

**Comment by timsuchanek** [comment]
> When I ran pnpm run setup, a few package.json files changed, is this normal? I've put these changes in a separate commit, so I can revert if necessary.

You can just commit that, updates of the pnpm-lock file are for example sometimes added.

> The test seems to be failing (during client generation) because of some Cloudfront issue. I don't think I understand the problem so I didn't try to debug too much.

That can happen if binaries are not yet fully released. A retry should fix that.

> That being said, I noticed that the version of the migrate package in its package.json is quite old, so I tried updating it to the latest dev, which didn't help (I don't think I should be doing it manually anyway). What can I do to fix this?

That is totally fine, it's linked in a workspace anyway.

**Pull request merged**